docs/Developers/Newcomers_Guide.rst

```diff
@@ -209,44 +197,10 @@ The easy issues that will help you get started are labeled as
 ``difficulty/newcomer`` and are only there to give you a glimpse of what
 it's like to work with us and what the workflow is like.
 
-The coala community creates a constant supply of ``newcomer`` issues available
-for people like you who are joining the organisation, so that you can learn
-how to contribute according to the coala organisation development guides,
-while doing a real task at the same time.
-
-coala does **not** follow the GitHub "flow" workflow, so be sure to read
-our guides carefully. You will need to use the ``git`` command line.
-During the newcomer process, we will help you conquer ``git``, so that you
-can become a part of our super efficient developer team.
-
-If you do not follow the coala workflow, our bots will get grumpy, and
-members of our community will try to help you in every way possible to
-get your Pull Request merged, the coala way.
-
-**Never** create a second Pull Request for your newcomer issue.
-Instead, if you have troubles on your first Pull Request, seek help from
-the manual or ask for guidance on `gitter chat <https://coala.io/chat>`_
-to reach a resolution for your troubles.
-
-Your Pull Request will **not** be accepted until it complies with coala
-standards, even if your solution is amazingly correct. If you ignore the
-bots or our developers, or if you close any Pull Request, we will have
-no option except to conclude that you have abandoned the coala newcomer
-process, and your amazing solution **will** be discarded.
-We may nudge you to finish your Pull Request, but eventually a maintainer
-will close incomplete Pull Request, and your newcomer issue will be returned
-to the available queue, ready for the next person learning the coala workflow.
-
 Now pick an issue which isn't assigned and which you would like to fix.
 Leave a comment that you would like to be assigned to the issue. This way
 we don't have multiple people working on the same issue at the same time.
-
-Remember that all newcomer issues should be very simple to fix, after you
-have understood the problem.
-Choose one of the newcomer issues that you really understand.
-If you have spent more than an hour debugging or coding, maybe your
-newcomer issue was incorrectly classified as a newcomer issue.
-Ask a developer to check if it should be reclassified as `low` difficulty.
```

Comment:
re-add

Reply:
I just don't understand how all of this is getting deleted. I never touch these parts of code. I just open "Newcomers_Guide.rst' file and make changes at one location only. How is all of this getting changed?

Reply:
I don't know how you changed all this but you can easily add all these again. Click on `Files changed` to see all your changes and just copy paste whatever unnecessary things you have removed.